users/views.py

```diff
@@ -4,20 +4,25 @@
 from django.views.decorators.csrf import csrf_exempt
 from django.core.mail import *
 from .models import ActivationMailer
-from campusdiscussbackend.settings import EMAIL_HOST_USER
+from campusdiscussbackend.settings_email import *
 # @csrf_exempt 
 # Do not use until you want it to POST
 
 # Create your views here.
-
-def send_email(request):
+'''
+The send_email function is an API for sending any email.
+In place of email_class, you can type in the class of email-type as
+mentioned in settings_email.py. An example class is given as ExampleEmail
+'''
+def send_email(request, email_class): 
```

Comment:
Are you sure this even works?

This is an API function. We don't need an API function to send an email - that should be done using a utility function.  The APIs (like sending a verification mail API) which require using that utility should call that utility function.

See [this question](https://stackoverflow.com/questions/58730090/location-for-app-wide-utils-file-in-a-django-project) and the link in the accepted answer.